docs/tutorials/lets_write_a_test.md

```diff
@@ -1,74 +1,82 @@
 ## The Task
 
-Consider if mcollectived incorrectly spawned a new process with every puppet agent run on Ubuntu 10.04.  We need an acceptance test to check that a new process is not spawned and to ensure that this issue does not regress in new builds.
+We will write a test to check if a package (specifically HTTPD) is installed and running. To do this we will write two files:
+1. `install.rb` - This file will install the package and start the service
+2. `mytest.rb` - This file will have our core tests that checks if the package is installed and running
 
-## Figure Out Test Steps
+Note: We will exclude Windows OS from our testing due to variation of package name.
+
+## Figure out steps
 
 What needs to happen in this test:
 
-* Install PE
-* Restart mcollective twice
-* Check to see if more than one mcollective process is running
+* Install and run HTTPD
+  * Install HTTPD if its not available on our SUT
+  * Start HTTPD service
+* Testing
+  * Test HTTPD is installed
+  * Test HTTPD service is running
 
 ## Create a host configuration file
     $ beaker-hostgenerator redhat7-64ma > redhat7-64ma.yaml
 
-## Install PE
-
-We prefer to install PE once and then run a set of tests, so PE installation should not be part of the actual acceptance test.
-
-    $ mkdir setup
-    $ cd setup
-    $ cat > install.rb << RUBY
-    test_name "Installing Puppet Enterprise" do
-    install_pe
-    RUBY
-    $ cd ..
+This command will generate a host file for our system under test (SUT). It will use vmpooler as hypervisor for the host. Please check out [this](https://github.com/puppetlabs/beaker/blob/master/docs/tutorials/quick_start_rake_tasks.md#hypervisor-argument) doc to learn more about hypervisors for beaker.
```

Comment:
I don't think that you want to link to that subsection of the quick start tasks, but link to the general hypervisors section: https://github.com/puppetlabs/beaker/tree/master/docs/how_to/hypervisors